R/dashboard.Rmd

```diff
@@ -1,28 +1,414 @@
+
+```
+
+### Total Current Month Trade Balance (Millions)
+```{r Current Month trade balance}
+valueBox(curtradebalance,
+         icon = 'fa-balance-scale',
+         color = "red")
+
+```
+
+
+```{r Calculate current year services import and export}
+valueInterested <- services[services$year == 2019, ]
+
+exportServices <- valueInterested$export
+importServices <- valueInterested$import
+serviceBalance <- exportServices - importServices
+
+```
+
+
+column {data-width=400}
+------------------------------------------------------------------
+### Services Export - 2019
+```{r}
+valueBox(exportServices,
+         icon = 'fa-balance-scale',
+         color = "green")
+```
+
+
+### Services Import - 2019
+```{r}
+
+valueBox(importServices,
+         icon = 'fa-balance-scale',
+         color = "blue")
+```
+
+
+### Services Trade Balance - 2019
+```{r}
+
+valueBox(serviceBalance,
+         icon = 'fa-balance-scale',
+         color = "green")
+```
+
+Column {data-width=650}
+-----------------------------------------------------------------------
+
+### Goods Export/Import/Balance by Year
+
+```{r goods exports/imports/balance by year interactive plot}
+trades %>%
+  group_by(Year) %>%
+  summarise(Export = sum(Export_mil),
+            Import = sum(Import_mil),
+            Balance = sum(TradeDef_mil), .groups="keep") %>%
+  ungroup() %>%
+  
+  plot_ly(x = ~Year,
+          y = ~Export,
+          color = "blue",
+          
+          type = "scatter",
+          mode = "lines+markers",
+          name = "Export") %>%
+  
+  add_trace(x = ~Year,
+            y = ~Import,
+            color = "red",
+            name = "Import") %>%
+  
+  add_trace(x = ~Year,
+            y = ~Balance,
+            color = "green",
+            name = "Balance") %>%
+  
+  layout(xaxis = list(title = "Year"), yaxis = list(title = 'Value'))
+```
+
+
+Column {data-width=650}
+-----------------------------------------------------------------------
+
+### Services Export/Import/Balance by Year
+
+```{r services exports/imports/balance by year interactive plot}
+trades %>%
+  group_by(Year) %>%
+  summarise(Export = sum(Service_exp),
+            Import = sum(Service_imp),
+            Balance = sum(Service_bal), .groups="keep") %>%
+  ungroup() %>%
+  
+  plot_ly(x = ~Year,
+          y = ~Export,
+          color = "blue",
+          type = "scatter", 
+          mode = "lines+markers",
+          name = "Export") %>%
+  
+  add_trace(x = ~Year,
+            y = ~Import,
+            color = "red",
+            name = "Import") %>%
+  
+  add_trace(x = ~Year,
+            y = ~Balance,
+            color = "green",
+            name = "Balance") %>%
+  
+  layout(xaxis = list(title = "Year"), yaxis = list(title = 'Value'))
+```
+
+
+
+Column {data-width=350}
+-----------------------------------------------------------------------
+
+### Trade Import Activities with 10 top countries overtime
+
+```{r animated graphic for top 10 countries by year for imports}

```

Comment:
Add `eval = FALSE` here to switch off when working on dashboard - it will greatly speed up loading